### PR TITLE
Add Destroy command

### PR DESCRIPTION
**Pull request by sungwoncho**
This is what I have so far on #192. 

It works, but does not output anything in the console as `generate` command does. Any suggestions?

### CODE REVIEW

**Comment by joneslee85** [comment]
@sungwoncho yes, i am happy with it outputing nothing atm. If you like, you could open a new PR to add in output text for generators/destroyers

File: lib/lotus/commands/destroy.rb [hunk 1]
```diff
@@ -0,0 +1,73 @@
...
+        @app_root ||= Pathname.new([@options[:path], app_name].join(::File::SEPARATOR))
+      end
+
+      def spec_root
+        @spec_root ||= Pathname.new('spec')
+      end
+
+      private
+
+        def controller_name
```
Comment (joneslee85) [inline lib/lotus/commands/destroy.rb]: our code style is not to indenting private methods

File: lib/lotus/commands/destroy.rb [hunk 2]
```diff
@@ -0,0 +1,73 @@
+module Lotus
+  module Commands
+    class Destroy
+
+      ACTION_SEPARATOR = /\/|\#/
+
+      attr_reader :name, :app_name
+
+      def initialize(type, app_name, name, env, cli)
```
Comment (joneslee85) [inline lib/lotus/commands/destroy.rb]: ideally, we would have many versions of destroyers, can we replicate same pattern like those of generators, that is having `destroyers/action.rb`

Comment (sungwoncho) [inline lib/lotus/commands/destroy.rb]: Good point. I will improve the PR using your suggestions.

**Comment by joneslee85** [comment]
@sungwoncho ping

**Comment by jodosha** [comment]
@sungwoncho Thanks for this PR. This makes sense, but before to merge it, we need to expand the scope of `destroy`, by including specular logic for `generate`. For instance, remove generated models.

Would you love to work on this? Thank you. :smile:

**Comment by joneslee85** [comment]
@t4deu please create a new PR

**Comment by t4deu** [comment]
Ok, @joneslee85, I will do in this weekend :smile:

**Pull request closed without merging**